### DISCUSSION TITLE
Place a bet modal window when clicked on event and event details when clicked on [Details] button [Suggestion]

### DISCUSSION THREAD

**Discussion by ztepler** [juster-finance/juster-app#19] [Ideas]
Предложение стимулировать пользователей делать ставки и поменять местами логику нажатия на карточку ивента и кнопки [Join]:
![image](https://user-images.githubusercontent.com/22714654/134205296-6e73d5d7-49b6-461b-affe-c945795617df.png)

1. При нажатии на любую часть карточки с ивентом показывать модельное окно со ставкой, которое сейчас показывается при нажатии на кнопку Join
2. Кнопку Join переименовать в Details
3. При нажатии на кнопку Details идти на страницу с деталями ивента, как сейчас происходит если нажать на карточку ивента

Таким образом число кликов для того чтобы сделать ставку будет ниже. Сценарий с просмотром деталей ивента мне кажется менее используемый чем сценарий добавления новой ставки.

Точно также предлагаю обсудить эту идею, если никто не против то реализовать.

**Comment by xdeq** [comment]
The `EventCard` has been redesigned
